app/services/solidus_square/webhooks/handlers/order_updated.rb

```diff
@@ -40,10 +40,14 @@ def order_amount
           order_info[:total_money][:amount] / 100.0
         end
 
+        def version
+          params[:data][:object][:order_updated][:version].to_i
+        end
+
         def create_square_payment!
           order.payments.find_or_create_by!(response_code: square_order_id) do |payment|
             payment.amount = order_amount
-            payment.source = ::SolidusSquare::PaymentSource.create!(token: square_order_id)
+            payment.source = ::SolidusSquare::PaymentSource.create!(token: square_order_id, version: version)
```

Comment:
This ticket aims to just create the version on the PaymentSource, we don't need to store it on the Source even because it is referring to the order.

Reply:
I did it, because otherwise my tests are failing as the version has a `presence: true`. And It would create an error running the tests without giving any version.

Reply:
What do you think to add a default value for it?

Reply:
Hmm 🤔 like 0? which is like nil? so maybe don't put the presence true?

Reply:
yes probably we don't need the presence validation